skimage/feature/_canny.py

```diff
@@ -265,6 +269,16 @@ def fsmooth(x):
 
         high_threshold = np.percentile(magnitude, 100.0 * high_threshold)
         low_threshold = np.percentile(magnitude, 100.0 * low_threshold)
+    else:
+        #
+        # Else if high_threshold and/or low_threshold are callables
+        # call them to determine the threshold value / image
+        #
```

Comment:
Do you mind removing the whitespace comments? I would also change `call them` to `apply them to the input image`.